go/developer/web.config

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <system.webServer>
+        <rewrite>
+            <rules>
+                <clear />
+
+                <!-- Links for Developer 10.0 onward -->
+
+                <rule name="/go/developer/X.0/help-keyboards" stopProcessing="true">
+                  <match url="^(10|11|12|13)\.0/help-keyboards" />
+                  <action type="Redirect" url="https://help.keyman.com/developer/keyboards/" />
+                </rule>
+                
+                <rule name="/go/developer/X.0/help-(mobile|packages)" stopProcessing="true">
+                  <match url="^(10|11|12)\.0/help-(mobile|packages)" />
```

Comment:
Similarly, we can assume the documentation will be available before release so let's include all versions here. We will always direct the help to the major version so the output redirect (hypothetical example 14.1 -> 14.0) is correct.